code/modules/power/lighting.dm

```diff
@@ -783,6 +783,8 @@
 		else
 			prot = 1
 
+		if (get_dist(src, user) > 1)
```

Comment:
please use the macro

Reply:
if you mean in_interact_range then I did. I tested it and it didn't work whatsoever. It still let me take out lights

Reply:
i actually meant `IN_RANGE`, but `in_interact_range` is indeed the correct one to use in this case.

Reply:
Same issue

Reply:
unable to reproduce. I literally changed the line locally and it worked fine. you probably forgot to negate it?